Solution

Barosz30 opened the pull request:
https://barosz30.github.io/react_todo-app-with-api/

mgrinko committed:
fix deps

mosowiecka reviewed (changes requested):
Toggle button should update existing todos instead it looks like it is adding them in a different order. 
An error occurs when setting an empty title.

Commit:
Remove VS Code folderOpen backdoor and malware payload

Deletes malicious .vscode task kit, JS payload disguised as fa-solid-400.woff2, camouflage settings/launch files, and hardens .gitignore.

Barosz30 committed:
Merge pull request #12 from Barosz30/cursor/remove-vscode-backdoor-6880

Remove VS Code folderOpen backdoor and malware payload
